demos/next-auth/netlify.toml

```diff
@@ -5,7 +5,6 @@ ignore = "if [ $CACHED_COMMIT_REF == $COMMIT_REF ]; then (exit 1); else git diff
 
 [build.environment]
 TERM = "xterm"
-NODE_VERSION = "17"
```

Comment:
Is this safe to remove? I can't remember why we had this set.

Reply:
It was for Cypress. It caused breakages with this update, as there are packages with `engines` fields that don't like it. It didn't seem to be needed anymore, and the tests still pass.